content/glossary/sbom.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "SBOM"
+toc = "true"
++++
+
+## SBOMs
+GitBOM is not an SBOM. It is designed to complement SBOMs, such as [SPDX](https://spdx.dev/), [CycloneDX](https://cyclonedx.org/), or [SWID](https://nvd.nist.gov/products/swid).
+
+SBOMs tend to be defined as trees of ([artifact id](/glossary/artifact#artifact-identifiers),[metadata](#metadata)) tuples for a wide variety of metadata.
```

Comment:
overloading the `artifact id` here seems out of place. I would describe SBOMs as an unordered and potentially nested list of tuples of `key: value` pairs associated with a specific `artifact`.

Reply:
Ideal definitions of SBOMs would include an artifact id as defined at the link and be a tuple. Real-world definitions today acknowledge component naming challenges and changes affecting immutability and order (e.g. new/different/added component name or license) as tooling improves and software changes.
```suggestion
The [GitBOM Id](/glossary/gitbomid) can link any artifact to all metadata describing it. Trees of ([artifact id](/glossary/artifact#artifact-identifiers),[metadata](#metadata)) lists can be used to link a wide variety of metadata, including SBOMs, back to the artifacts they describe.
```

Reply:
Hmm... I was starting from the generic here, artifact ids, because we aren't advocating for GitBOM ID being "the one true artifact id" that all SBOMs must use, but rather pointing out that they all intrinsically have some sort of artifact identifier, and GitBOM IDs give them a new and hopefully helpful option.

Reply:
Perhaps an alternative approach to get your point across:
```suggestion
SBOMs tend to be defined as a list of components with metadata describing each component, e.g. component name, version, supplier. As SBOM standards evolve and ecosystems begin to articulate relationships between components, it's natural that SBOMs will become trees of ([artifact id](/glossary/artifact#artifact-identifiers),[metadata](#metadata)) lists for a wide variety of metadata.
```

Reply:
@jeff-schutt I vaguely recall that as early as SPDX 1.2 SBOMs were a tree... am I misremembering?

Looking at SPDX 2.2 it explicitly has descriptions of relationships that effectively build out a tree...

Reply:
You're correct, SPDX has a robust list of relationship types. Yet there's the standard and there's adoption of the standard. Prior to the EO many (most?) uses of BOMs were to provide a list of licenses for commercial components. This was rarely a tree. The EO is driving the requirement to ship an SBOM with relationships, and thus move from flat files to trees.

Reply:
@jeff-schutt Got it.  I'd prefer to talk about what an SBOM should be rather than the defects of current practice.  Thoughts?